### ISSUE TITLE
ImageTexture.update() doesn't seem to work

### ISSUE THREAD

**Issue by Zylann** [godotengine/godot#63131]
### Godot version

5a6b13b8bb128a0e10cccab8f212b1ed15cea425

### System information

Windows 10 64 bits GLES3 GeForce GTX 1060 6GB/PCIe/SSE2

### Issue description

The documentation of `ImageTexture.update()` says:
```
Note: The texture has to be initialized first with the create_from_image() method before it can be updated. The new image dimensions, format, and mipmaps configuration should match the existing texture's image configuration, otherwise it has to be re-created with the create_from_image() method.
```
But in my case it keeps printing an error even if the texture was initialized with `create_from_image`

### Steps to reproduce

```gdscript
extends Node

var _image : Image
var _texture : ImageTexture

func _ready():
	_image = Image.new()
	_image.create(100,100, false, Image.FORMAT_RGB8)
	
	_texture = ImageTexture.new()
	_texture.create_from_image(_image)

func _process(delta):
	_texture.update(_image)
```
```
E 0:00:01:0593   _process: Texture is not initialized.
  <C++ Error>    Condition "texture.is_null()" is true.
  <C++ Source>   scene\resources\texture.cpp:204 @ ImageTexture::update()
  <Stack Trace>  main.gd:16 @ _process()
```

### Minimal reproduction project

[ImageTextureUpdate.zip](https://github.com/godotengine/godot/files/9128699/ImageTextureUpdate.zip)

**Comment by KoBeWi** [comment]
Outdated description.

**Comment by Zylann** [comment]
ARGH I got played twice by the same thing
```
_texture.create_from_image(_image)
```
`create_from_image` was changed to become static. There was no warnings or errors whatsoever about calling it as a method. So doing this works instead:
```
_texture = ImageTexture.create_from_image(_image)
```
@KoBeWi I already read the doc about `update`, it was clear to me already. Even with your PR that wouldn't have helped here cuz I couldn't notice `create_from_image` was static xD
